<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDisplay_Front_Camera"/>
    <addaction name="actionDisplay_Side_Camera"/>
    <addaction name="actionDisplay_Top_Camera"/>
    <addaction name="actionDisplay_Shapes"/>
    <addaction name="actionDisplay_Histograms"/>
    <addaction name="actionDisplay_Masks"/>
    <addaction name="actionDetect_Color_Chips"/>
   </widget>
   <widget class="QMenu" name="menuProperties">
    <property name="title">
     <string>Properties</string>
    </property>
    <addaction name="actionModify_Region_of_Intrest"/>
    <addaction name="actionModify_Mask_Properties"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuProperties"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionImport_Images"/>
   <addaction name="actionDetect_Cameras"/>
   <addaction name="actionLoad_Cameras"/>
   <addaction name="actionActivate_Cameras"/>
   <addaction name="actionOpen_Camera"/>
   <addaction name="actionDetect_Color_Chips"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDisplay_Front_Camera"/>
   <addaction name="actionDisplay_Side_Camera"/>
   <addaction name="actionDisplay_Top_Camera"/>
   <addaction name="actionDisplay_Masks"/>
   <addaction name="actionDisplay_Shapes"/>
   <addaction name="actionDisplay_Histograms"/>
  </widget>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/Images/CreateNewProject.png</normaloff>:/icons/Images/CreateNewProject.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Meta+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/Images/OpenNewProject.png</normaloff>:/icons/Images/OpenNewProject.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Meta+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Capture Plants</string>
   </property>
   <property name="shortcut">
    <string>Meta+C</string>
   </property>
  </action>
  <action name="actionDisplay_Front_Camera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/Images/FrontView.png</normaloff>:/icons/Images/FrontView.png</iconset>
   </property>
   <property name="text">
    <string>Display Front Camera</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDisplay_Side_Camera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Display Side Camera</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDisplay_Top_Camera">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/Images/TopView.png</normaloff>:/icons/Images/TopView.png</iconset>
   </property>
   <property name="text">
    <string>Display Top Camera</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionModify_Region_of_Intrest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modify Region of Interest</string>
   </property>
  </action>
  <action name="actionModify_Mask_Properties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modify Mask Properties</string>
   </property>
  </action>
  <action name="actionActivate_Cameras">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/Images/ActivateCameras.png</normaloff>:/icons/Images/ActivateCameras.png</iconset>
   </property>
   <property name="text">
    <string>Activate Cameras</string>
   </property>
  </action>
  <action name="actionDisplay_Shapes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Display Shapes</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDisplay_Histograms">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Display Histograms</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDisplay_Masks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Display Masks</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDetect_Color_Chips">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Detect Color Chips</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDetect_Cameras">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/Images/DetectCameras.png</normaloff>:/icons/Images/DetectCameras.png</iconset>
   </property>
   <property name="text">
    <string>Detect Cameras</string>
   </property>
  </action>
  <action name="actionLoad_Cameras">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/icons/Images/LoadCameras.png</normaloff>:/icons/Images/LoadCameras.png</iconset>
   </property>
   <property name="text">
    <string>Load Cameras</string>
   </property>
  </action>
  <action name="actionImport_Images">
   <property name="text">
    <string>Import Images</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
